src/components/dropdown/Options.vue

```diff
@@ -53,10 +53,7 @@ export default {
     },
 
     getText(option) {
-      if (this.optionText) {
-        return option[this.optionText];
-      }
-      return option;
+      return typeof option === 'object' && this.optionText ? option[this.optionText] : option;
```

Comment:
I think this change is not needed as by default `optionText` is `null` so it doesn't make any difference for primitive values as fallbacks to `option` anyway.